client/src/components/image.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Image = ({ image, myClass }) => {
```

Comment:
This looks pretty good, though do you think the component name could be more descriptive? This seems to be a collection of images, rather than just one. Total nitpick, but that's all I can do when the code is solid!